crates/wallet/src/wallet/mod.rs

```diff
@@ -2265,31 +2265,7 @@ impl Wallet {
     ///
     /// After applying updates you should persist the staged wallet changes. For an example of how
     /// to persist staged wallet changes see [`Wallet::reveal_next_address`].
-    #[cfg(feature = "std")]
-    #[cfg_attr(docsrs, doc(cfg(feature = "std")))]
     pub fn apply_update(&mut self, update: impl Into<Update>) -> Result<(), CannotConnectError> {
-        use std::time::*;
-        let now = SystemTime::now()
-            .duration_since(UNIX_EPOCH)
-            .expect("time now must surpass epoch anchor");
-        self.apply_update_at(update, now.as_secs())
-    }
-
-    /// Applies an `update` alongside a `seen_at` timestamp and stages the changes.
-    ///
-    /// `seen_at` represents when the update is seen (in unix seconds). It is used to determine the
-    /// `last_seen`s for all transactions in the update which have no corresponding anchor(s). The
-    /// `last_seen` value is used internally to determine precedence of conflicting unconfirmed
-    /// transactions (where the transaction with the lower `last_seen` value is omitted from the
-    /// canonical history).
-    ///
-    /// Use [`apply_update`](Wallet::apply_update) to have the `seen_at` value automatically set to
-    /// the current time.
-    pub fn apply_update_at(
```

Comment:
Hey removing this would be a breaking 1.x change so we need to discuss again how to release this change.

Reply:
I guess we can deprecate it and state that the timestamp here does nothing. 

However changing a method to require a feature flag is also breaking.

Reply:
After the [lib call](https://discord.com/channels/753336465005608961/753336465005608965/1344807550217883688), the consensus was to put `bdk_wallet` on a different release schedule to the other crates. This PR is modified to not affect `bdk_wallet`.